server/routes/publishers.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const publisherController = require('../controllers/publisher.controller');
+
+const router = express.Router();
+
+router.get('/', (req, res) => {
```

Comment:
More concise vesrsion
```suggestion
router.get('/', publisherController.select);
```